interview-questions.md

```diff
@@ -7,90 +7,111 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would you use the method `super()`?
 
-  Your answer:
+  Your answer: super() is used to gain access to functions in a parent class
 
-  Researched answer:
+  Researched answer: By calling the super() method in the constructor method, we call the parent's constructor method and gets access to the parent's properties and methods.
 
 
 
 2. What is the DOM? What is the virtual DOM?
 
-  Your answer:
+  Your answer: Document Object Model, and virtual DOM sits somewhere between our code and the real DOM, but I'm not really confident what they are
```

Comment:
No worries! You got it right! The virtual DOM is React specific, allowing us and our users to make updates and changes to our DOM without refreshing the entire page.